.devcontainer/Dockerfile

```diff
@@ -42,124 +45,60 @@ RUN echo "Install general purpose packages" && \
-        update-alternatives --install /usr/bin/clang++ clang++ /usr/lib/llvm-11/bin/clang++ 10 && \
-        update-alternatives --install /usr/bin/clang-format clang-format /usr/lib/llvm-11/bin/clang-format 10 && \
-        update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/lib/llvm-11/bin/clang-tidy 10 && \
-        update-alternatives --install /usr/bin/clang-apply-replacements clang-apply-replacements /usr/lib/llvm-11/bin/clang-apply-replacements 10
-
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y clangd-12
-
-# Install bazel
-WORKDIR /usr/sbin
-RUN wget --progress=dot:giga https://github.com/bazelbuild/bazelisk/releases/download/v1.10.0/bazelisk-linux-amd64 && \
-    chmod +x bazelisk-linux-amd64 && \
-    ln -s /usr/sbin/bazelisk-linux-amd64 /usr/sbin/bazel
+        virtualenv=20.0.17-1ubuntu0.4 && \
+    gem install fpm && \
+    update-alternatives --install /usr/bin/clang clang /usr/lib/llvm-11/bin/clang 10 && \
+    update-alternatives --install /usr/bin/clang++ clang++ /usr/lib/llvm-11/bin/clang++ 10 && \
+    update-alternatives --install /usr/bin/clang-format clang-format /usr/lib/llvm-11/bin/clang-format 10 && \
+    update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/lib/llvm-11/bin/clang-tidy 10 && \
+    update-alternatives --install /usr/bin/clang-apply-replacements clang-apply-replacements /usr/lib/llvm-11/bin/clang-apply-replacements 10
 
 RUN echo "Install 3rd party dependencies" && \
     apt-get update && \
-    echo "Install CMake" && \
-    apt-get -y install cmake && \
     echo "Install FMT lib requirements" && \
     apt-get -y install libunwind8-dev libelf-dev libdwarf-dev bzip2 && \
     echo "Install Folly requirements" && \
-    apt-get -y install libboost-all-dev libevent-dev libdouble-conversion-dev \
-    libgoogle-glog-dev libgflags-dev libiberty-dev liblz4-dev liblzma-dev \
-    libsnappy-dev binutils-dev libjemalloc-dev libssl-dev pkg-config libunwind-dev && \
+    apt-get -y install \
+    liblz4-dev liblzma-dev \
+    libsnappy-dev binutils-dev libjemalloc-dev pkg-config libunwind-dev && \
     echo "Install check for test support" && \
     apt-get -y install check && \
     echo "Install gtest for test support" && \
     apt-get -y install libgtest-dev && \
-    echo "Install FreeDiameter requirements" && \
-    apt-get -y install libsctp1 libsctp-dev libgcrypt-dev \
-    bison flex libidn11-dev && \
-    echo "Install libgtpnl requirements" && \
-    apt-get -y install libmnl-dev && \
     echo "Install Nettle requirements" && \
-    apt-get install -y libgoogle-glog-dev libconfig-dev libxml2-dev \
+    apt-get install -y libxml2-dev \
     libyaml-cpp-dev nlohmann-json3-dev && \
-    echo "Install ZeroMQ" && \
-    apt-get install -y libczmq-dev && \
-    echo "Install dependencies for Sentry Native" && \
-    apt-get install -y libcurl4-openssl-dev
+    echo "GRPC and it's dependencies" && \
+    apt-get install -y grpc-dev
```

Comment:
- There is inconsistency regarding the apt-get parameters here. They should be aligned for the installations in this file.
- Above there is a one-package-per-line philosophy followed, which is much more readable. I think we should also follow this approach here.